### DISCUSSION TITLE
Hysteria Optimization

### DISCUSSION THREAD

**Discussion by private-number** [apernet/hysteria#641] [Q&A]
Hi guys

Consider me as newbie and ELI5 

I was looking to use hysteria but I need help how to find best "recv_window_conn" and "recv_window" and "up_mbps" and  "down_mbps"

I think up/down numbers are about what is my download speed when i download/upload directly so i need to know if i'm wrong or right about it.

about "recv_window_conn" and "recv_window" i read that it's different for everyone and I need to find optimum number in order to reach hysteria full capability and speed. but I don't have any idea how to find it.

Thanks.

**Comment by chika0801** [comment]
https://github.com/apernet/hysteria/issues/644#issuecomment-1669753890
